src/general/WorldGenerationSettings.cs

```diff
@@ -32,14 +34,22 @@ public WorldGenerationSettings()
 
     public enum LifeOrigin
     {
+        [Description("LIFE_ORIGIN_VENTS")]
```

Comment:
What are these? I don't remember seeing these before...

Reply:
An approach for translating enum values (see the new methods at the end of this file). There's an example in `SaveInformation` already.

Reply:
I see there's already `Description` in the translation extraction script so I suppose this is fine, though this is one more thing for new people to understand about the translation system...